src/gbd_mapping_generator/coverage_gap_builder.py

```diff
@@ -93,7 +93,8 @@ def make_coverage_gap(name, gbd_id, distribution, restrictions, levels=None, aff
 
 def make_coverage_gaps(coverage_gap_data):
     out = "coverage_gaps = CoverageGaps(\n"
-    for name, gbd_id, distribution, restrictions, levels, affected_causes, affected_risks in coverage_gap_data:
+    for data in coverage_gap_data:
```

Comment:
Is there a reason you want to do this on two lines?  Just curious.  I don't mind either way.

Reply:
I think I added it during the debugging stage, since it was a huge tuples and wasn't clear to see. I can remove that